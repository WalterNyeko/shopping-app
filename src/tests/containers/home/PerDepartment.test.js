import React from "react";
import { shallow, mount } from "enzyme";
import PerDepartment from "../../../components/home/PerDepartmentPage";

describe("PerDepartment", () => {
  const props = {
    match: {
      params: {
        departmentId: 1
      }
    }
  };
  const wrapper = shallow(<PerDepartment {...props} />);
  it("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
