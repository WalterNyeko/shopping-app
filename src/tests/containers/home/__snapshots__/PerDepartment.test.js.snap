// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PerDepartment should render correctly 1`] = `ShallowWrapper {}`;
